src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
+    return days * 40 - 20;
+  } else {
+    return days * 40 - 50;
```

Comment:
```suggestion
  return days * 40 - 50;
```
Можна без else, адже кожна попередня умова повертає щось